src/fields/table_function.cc

```diff
@@ -30,12 +30,13 @@ const it::Tuple & TableFunction<Value>::get_input_type_val()
 	 * to independent value. It needs only replace type of 't' with generic type
 	 * (Input::Type::Parameter).
 	 */
+    typedef FieldValue<3>::TensorFixed TensorValue;
     return it::Tuple("IndependentValue", "Value of Field for time variable.")
                                        //"Value of Field for independent variable."
         .declare_key("t", TimeGovernor::get_input_time_type( 0.0 ), it::Default::obligatory(),
                                     "Time stamp." )
                                   //"Independent variable of stamp."
-		.declare_key("value", Value::get_input_type(), it::Default::obligatory(),
+		.declare_key("value", TensorValue::get_input_type(), it::Default::obligatory(),
```

Comment:
replace  TensorValue::get_input_type by a function out of FieldValue class template to have just one input type function for all shapes.